public/scripts/world-info.js

```diff
@@ -1867,7 +1875,8 @@ function displayWorldEntries(name, data, navigation = navigation_option.none, fl
                     Trigger %
                 </small>
             </div>`;
-            const blocks = page.map(entry => getWorldEntry(name, data, entry)).filter(x => x);
+            const mappedEntryBlocksPromises = page.map(async entry => await getWorldEntry(name, data, entry)).filter(x => x);
```

Comment:
Doing async/await in map callback doesn't make sense. Doing it below with `Promise.all` is correct.

Reply:
Question is, does it *hurt* though? I don't like calling an async function in a non-async context unless explicitly required there.